src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -151,60 +152,49 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
             await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
-            _componentModel = (IComponentModel)await GetServiceAsync(typeof(SComponentModel)).ConfigureAwait(true);
```

Comment:
Tagging @CyrusNajmabadi only because he's touching this area too.

Reply:
Oh but this is the other package. Goodness I hate this code.

Reply:
we could likely move the component model acquisition up to the base type since both subclasses seem to need it immediately, and both call base.InitializeAsync.